airbrake/notifier.py

```diff
@@ -92,24 +108,27 @@ def __repr__(self):
 
     @property
     def context(self):
-        """The python, os, and app environment context."""
+        """System, application, and user data to make more sense of errors."""
         if not self._context:
-            self._context = {}
-            # python
             version = platform.python_version()
-            self._context.update({'language': 'Python %s' % version})
-            # os
             plat = platform.platform()
```

Comment:
I don't think I understand what you mean here. Using `_context` lets us avoid calls to `platform.platform()` and `platform.python_version()`?